src/test/java/bio/terra/aws/resource/discovery/EnvironmentDiscoveryTestBase.java

```diff
@@ -268,4 +268,84 @@ public void addFieldBeforeSchemaUpdateTestLogic(EnvironmentDiscovery environment
       throws IOException {
     noLandingZonesTestLogic(environmentDiscovery);
   }
+
+  private void expectMissingLandingZoneOptionals(
+      LandingZone expectedLandingZone, LandingZone landingZone) {
+    Assertions.assertTrue(landingZone.getApplicationVpcId().isEmpty());
+    Assertions.assertTrue(landingZone.getApplicationVpcId().isEmpty());
```

Comment:
was this second one meant to be the private subnet id?

Reply:
Thank you!